pkg/stubs/stubs.go

```diff
@@ -51,7 +51,7 @@ func Create(path string, testType config.FuzzTestType) error {
 			}
 			fileNameExtension, found := config.TestTypeFileNameExtension(testType)
 			if !found {
-				return errors.WithStack(err)
+				return errors.Errorf("unknown file extension %s", testType)
```

Comment:
`testType` is not a file extension. Also, I think not finding a file extension here warrants a panic because it's clearly a programming error.

```suggestion
				panic(fmt.Sprintf("no file name extension found for test type %s", testType))
```